sys-libs/mesa/patches/mesa-19.1.7.patchset

```diff
@@ -0,0 +1,414 @@
+From f632aac9385cfe02f5bfe762d93e60b5b46f2c77 Tue Sep 17 00:00:00 2019
+From: Ken Mays <kmays2000@gmail.com>
```

Comment:
AFAIk your patchset based on he 17.x patchset, if so,be respectful and do not takeover their work.

Reply:
We don't use that authorship information anywhere, and this is the default behavior of --extract-patchset. I don't see why we should nitpick on this. There are enough real problems to fix here.

Reply:
> We don't use that authorship information anywhere, and this is the default behavior of --extract-patchset. I don't see why we should nitpick on this. There are enough real problems to fix here.

IMO regarding upstream, this makes things more complex for no good reason. This patch is probably useless anyway.

Reply:
@korli - The BGLView patch was needed from the 'Flurry' and 'Gravity' screensaver bug fix.
The other patches fix the native build issue. Mainly, 2-4 patches remain now which I upstreamed for Mesa-dev review.

Reply:
Looks fun indeed: https://gitlab.freedesktop.org/mesa/mesa/merge_requests/2062